finality-aleph/src/network/gossip/mock.rs

```diff
@@ -112,11 +112,12 @@ impl MockRawNetwork {
         MockEventStream(rx)
     }
 
-    pub fn emit_event(&mut self, event: MockEvent) {
-        for sink in &*self.event_sinks.lock() {
-            sink.unbounded_send(event.clone()).unwrap();
-        }
-    }
+    // TODO: integrate new Substrate network API properly
+    // pub fn emit_event(&mut self, event: MockEvent) {
```

Comment:
Don't leave commented code in here. You can keep the "TODO" somewhere around here, although I would love if it referred to a specific task (but yeah, jira outage...).

Reply:
done